### DISCUSSION TITLE
Where is documentation for creating themes?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [johnlindquist/kit#1407] [Q&A]
I did some looking up but didn't find anything, other than creating one from the theme selector. I want a list of all the possible options.

**Comment by johnlindquist** [comment]
If you look in `~/.kit/themes/`, you'll see the .json files for themes with all the options.

In your `~/.kenv/.env`, you can also set the app fonts:

```sh
KIT_TERM_FONT=Geist Mono
```

KIT_MONO_FONT, KIT_SANS_FONT, KIT_SERIF_FONT are also available, but never really tested.

If there's something else you're looking to customize, I'm open to feedback

**Reply by YousufSSyed** [reply]
So these are the options?

```json
{
  "name": "Custom Theme",
  "foreground": "#ffffff",
  "accent": "#fbbf24",
  "ui": "#343434",
  "background": "#000000",
  "opacity": "0.85"
}
```

Would it be possible to completely control the look with CSS, that I could apply to any of the elements?

**Reply by johnlindquist** [reply]
Do you have a specific end goal? There's not really a system in place for that, so it might get really hacky depending on what you want to do

**Reply by YousufSSyed** [reply]
What UI Framework does Scriptkit use? I'd like to be able to use dev tools to inspect the elements and write CSS, like its an Electron app.

**Reply by johnlindquist** [reply]
It is Electron, you can use dev tools from the Tray icon -> Debug -> Open Dev Tools (unless you're on Linux, then you'll have to press whatever the Linux dev tools shortcut is since Linux doesn't support mouse events on tray icons)

**Reply by YousufSSyed** [reply]
Could I inspect the palette UI with it, and write permanent CSS for it?
